routes/projects/read.js

```diff
@@ -44,6 +44,10 @@ module.exports = function(config, req, res) {
       return;
     }
 
+    projects = projects.sort(function(project1, project2) {
```

Comment:
`sort` is odd in that it does the changes *in place* so you can just do `projects.sort(...)` and it will do what you want.